english/jacket/marshmallow-jacket.md

```diff
@@ -0,0 +1,3 @@
+After getting out of the leather jacket, you decide to try your luck at creating a new jacket out of marshmallows. Your hope is to make this one less restrictive.  You get started right away melting and cooling marshmallows in to the form of a jacket.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.